package.json

```diff
@@ -59,7 +59,7 @@
         "@octokit/rest": "^16.34.0",
         "@patternfly/quickstarts": "^1.4.1-rc.1",
         "@patternfly/react-core": "^4.147.0",
-        "@patternfly/react-icons": "^4.10.2",
+        "@patternfly/react-icons": "^4.53.16",
```

Comment:
Can you update the peer dependency of the components package? Consumers with older PF icons version will not have the icon available and it can cause issues.

Reply:
@Hyperkid123 Done, please take a look

Reply:
Please run `npm install` in the repository root. The lockfile is still not updated.

Reply:
@Hyperkid123 ran npm install and pushed the update